docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1221,6 +1203,456 @@ thor:
       </sect3>
     </sect2>
 
+    <sect2 id="HPCCContainer_Certificates">
+      <title>Certificates</title>
+
+      <para>HPCC Systems containerized deployments support comprehensive
+      certificate management through Helm chart configuration. This includes
+      support for multiple certificate issuers, domain management, and
+      advanced certificate features for secure communication between
+      components.</para>
+
+      <para>The certificate system integrates with cert-manager to provide
+      automated certificate generation and renewal for both internal (local)
+      and external (public/remote) communications.</para>
+
+      <sect3 id="Container_Certificates">
+        <title>Enable Certificates</title>
+
+        <para>The certificates section can be used to enable the cert-manager
```

Comment:
PV
The certificates section can be used to enable the cert-manager
s/b 
Use the certificates section to enable cert-manager

Reply:
This was not addressed